src/DynamoCoreWpf/ViewModels/Core/StateMachine.cs

```diff
@@ -503,10 +503,10 @@ internal bool HandleLeftButtonDown(object sender, MouseButtonEventArgs e)
 
                     eventHandled = true; // Mouse event handled.
                 }
-                else if (this.currentState == State.None)
+                else if (currentState == State.None)
                 {
                     // Record the mouse down position.
-                    IInputElement element = sender as IInputElement;
+                    var element = sender as IInputElement;
```

Comment:
C# coding conventions say to use implicit typing for local variables when the type of the variable is obvious from the right side of the assignment, or when the precise type is not important. And it is the case